pdns/README-dnsdist.md

```diff
@@ -857,6 +862,12 @@ The second one is the maximum lifetime of an entry in the cache, the third one i
 the minimum TTL an entry should have to be considered for insertion in the cache,
 the fourth one is the TTL used for a Server Failure or a Refused response. The last
 one is the TTL that will be used when a stale cache entry is returned.
+For performance reasons the cache will pre-allocate buckets based on the maximum number
+of entries, so be careful to set the first parameter to a reasonable value. Something
```

Comment:
Sure, but since it depends on the amount of memory you are willing to allocate to the cache, I'm not sure what to indicate here?